sdk/communication/azure-communication-sms/src/main/java/com/azure/communication/sms/SmsAsyncClient.java

```diff
@@ -97,9 +95,12 @@
      * @param message message to send to recipient.
      * @return response for a successful send Sms request.
      */
-    @ServiceMethod(returns = ReturnType.COLLECTION)
-    public PagedFlux<SmsSendResult> send(String from, Iterable<String> to, String message) {
-        return send(from, to, message, null, null);
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<Iterable<SmsSendResult>> send(String from, Iterable<String> to, String message) {
+        return sendWithResponse(from, to, message, null)
+            .flatMap((Response<Iterable<SmsSendResult>> response) -> {
+                return Mono.just(response.getValue());
+            });
```

Comment:
You can use a map here instead of flatMap() - `map(response -> response.getValue())`?